packages/@n8n/nodes-langchain/nodes/chains/InformationExtractor/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from 'zod';
+import type { AttributeDefinition } from './types';
+
+function makeAttributeSchema(attributeDefinition: AttributeDefinition, required: boolean = true) {
+	let schema: z.ZodTypeAny;
+
+	if (attributeDefinition.type === 'string') {
+		schema = z.string();
+	} else if (attributeDefinition.type === 'number') {
+		schema = z.number();
+	} else if (attributeDefinition.type === 'boolean') {
+		schema = z.boolean();
+	} else if (attributeDefinition.type === 'date') {
+		schema = z.string().date();
+	} else if (attributeDefinition.type === 'time') {
+		schema = z.string().time();
+	} else if (attributeDefinition.type === 'datetime') {
```

Comment:
These are not valid according to your type definition, I guess?

Reply:
Yep, it was there at first, but I found it too brittle while testing the node and decided to remove for now